### PR TITLE
Qianfan/arguments pass through

### PR DESCRIPTION
**Pull request by Dobiichi-Origami**

### CODE REVIEW

File: libs/langchain/langchain/chat_models/baidu_qianfan_endpoint.py [hunk 1]
```diff
@@ -180,7 +182,7 @@ def _default_params(self) -> Dict[str, Any]:
             "penalty_score": self.penalty_score,
         }
 
-        return {**normal_params, **self.model_kwargs}
```
[the remark this answers is not in the document]
Comment (Dobiichi-Origami) [reply libs/langchain/langchain/chat_models/baidu_qianfan_endpoint.py:183]: we decide to remove this attribute due to its redundancy in functionality and make **kwargs in calling to take place of it :)

[the remark this answers is not in the document]
Comment (Dobiichi-Origami) [reply libs/langchain/langchain/chat_models/baidu_qianfan_endpoint.py:183]: OG self.model_kwargs has become self.init_kwargs which initializes underlying qianfan component dependency

[the remark this answers is not in the document]
Comment (Dobiichi-Origami) [reply libs/langchain/langchain/chat_models/baidu_qianfan_endpoint.py:183]: after conversation between me and developers of these module

**Comment by Dobiichi-Origami** [comment]
@jarib @sbusso

**Comment by Dobiichi-Origami** [comment]
@baskaryan Is there any question to be solved or answered? PTAL

**Comment by Dobiichi-Origami** [comment]
@efriis PLEASE TAKE A LOOK WE NEED FEEDBACK

**Comment by efriis** [comment]
Hey @Dobiichi-Origami ! This is a breaking change without justification of why it's necessary, so I'll close it. If you want to reopen, I would recommend including a PR description and trying to avoid breaking changes for any existing users!

**Pull request closed without merging**